### PR TITLE
Add isValidAddress

### PR DESCRIPTION
**Pull request by clark800**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -31,3 +31,14 @@ module.exports = apiFactory({
     K256Seed: {version: FAMILY_SEED}
   }
 });
+
+exports.isValidAddress = function(address) {
```
Comment (sublimator) [inline src/index.js]: Maybe you just want to change AccountID to Address everywhere? There's `validateX` functions added to the api for all of the declaratively configured versions, adding expectedLength while we are at it.  We can change that to isValidX, which is a more suitable name for a boolean returning function, indeed indeed.

**Comment by clark800** [comment]
I think we can close this pull request because it looks like the same functionality is added in: https://github.com/ripple/ripple-address-codec/pull/2

**Comment by sublimator** [comment]
You aren't admin here? I thought I added the ripple lib admin team?

**Comment by clark800** [comment]
I am admin, I just wanted to make sure because you didn't mention it.

**Pull request closed without merging**